cmd/krew/cmd/search.go

```diff
@@ -82,9 +84,21 @@ Examples:
 
 		var searchResults []string
 		if len(args) > 0 {
+			// find by name
 			matches := fuzzy.Find(strings.Join(args, ""), pluginCanonicalNames)
+			searchResultsMap := make(map[int]bool)
```

Comment:
I wonder if we should stop sorting by name (and sort by relevance/how close it is to an exact match) instead when there's a query specified.

Reply:
The name is just one word without space, so the fuzzy search did not check the `match.Score`.
There are word spaces when search by description, so checked the `match.Score > 0`, to ignore the low score match.